.clang-tidy

```diff
@@ -63,3 +63,9 @@ CheckOptions:
       key: readability-identifier-naming.MethodIgnoredRegexp,
       value: '^classof$',
     }
+  - {
+      # This erroneously fires in C++20 mode with LLVM 16 clang-tidy, likely due
+      # to: https://github.com/llvm/llvm-project/issues/61969
```

Comment:
```suggestion
      # This erroneously fires in C++20 mode with LLVM 16 clang-tidy, due to:
      # https://github.com/llvm/llvm-project/issues/46097
```

Reply:
Are  you sure 46097 is right? That seems about `error: invalid case style for template parameter 'expr-type'`, not `clang-diagnostic-builtin-macro-redefined` or suppressed warnings being treated as errors.

Reply:
The error that's being suppressed here is for a template parameter named `expr-type`, which is 46097. 61969 is the bug number corresponding to the patch instead.